src/BenchmarkDotNet/Toolchains/CoreRt/Generator.cs

```diff
@@ -0,0 +1,160 @@
+using BenchmarkDotNet.Loggers;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Toolchains.CsProj;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using BenchmarkDotNet.Extensions;
+using BenchmarkDotNet.Portability;
+
+namespace BenchmarkDotNet.Toolchains.CoreRt
+{
+    /// <summary>
+    /// generates new csproj file for self-contained .NET Core RT app
+    /// based on https://github.com/dotnet/corert/blob/master/Documentation/how-to-build-and-run-ilcompiler-in-console-shell-prompt.md and https://github.com/dotnet/corert/tree/master/samples/HelloWorld#add-corert-to-your-project
```

Comment:
@adamsitnik could you use links to specific revisions instead of `blob/master`?

Reply:
sure, good idea!